src/stories/components/FileUpload/FileUpload.js

```diff
@@ -0,0 +1,59 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import UploadLogo from '../../../images/file_upload-white-18dp.svg';
+import { Button } from '../Button/Button';
+
+/**
+ * Component for File Upload element.
+ *
+ * @component
+ * @param {func} onClick File Upload click function
+ * @return {object} (
+ *   <FileUpload onClick={onClick} />
+ * )
+ */
+export const FileUpload = ({ onClick }) => {
+  const buttonText = (
+    <div className="upload-icon">
+      <p>Upload <img src={UploadLogo} /></p>
+    </div>
+  );
+
+  const showFileName = () => {
+    if (document.getElementById('file-upload__input')) {
+      const name = document.getElementById('file-upload__input');
+      const selectedFile = name.files.item(0).name;
+      document.getElementById('file-upload__selected').innerHTML = selectedFile;
+    }
+  };
+
+  return (
+    <div className="file-upload-container">
```

Comment:
**Nit-pick Alert**: This class should called `file-upload` since it's the component